news/2017-07-22-beatmap-spotlight-june-2017.md

```diff
@@ -0,0 +1,106 @@
+
+[![](/wiki/shared/news/2017-07-22-beatmap-spotlights-june-2017/teriquma.jpg)](https://osu.ppy.sh/s/615774)
+
+**[Charlotte](https://osu.ppy.sh/u/3686901)** sure has been present in the osu!taiko spotlights these last months, and rightfully so! Once again providing the ranked section with a great mapset, featuring as always a flawless structure and a perfect 1/6 snapping usage in the Inner Oni, it grealy deserves the attention of taiko players this month!
+
+[![](/wiki/shared/news/2017-07-22-beatmap-spotlights-june-2017/blinded-in-chains.jpg)](https://osu.ppy.sh/s/536955)
+
+According mapping experience, **[gaston_2199](https://osu.ppy.sh/u/5938161)** is the least experienced osu!taiko mapper among nominated mappers, but it doesn't impact the quality of this map. The map contains quite solid structure with common pattern. However, it doesn't make the map become boring. He uses different combination of patterns but same structure to distinguish the difference of melody precisely. Beside that, the highlight of this map is streams. He also adopts similar idea, and hitsounds fit guitar, drum and even vocal pretty well. In conclusion, the map is well structured and could be a good model to learn how to map taiko map on this kind of song.
+
+[![](/wiki/shared/news/2017-07-22-beatmap-spotlights-june-2017/we-are-number-one.jpg)](https://osu.ppy.sh/s/547301)
+
+Following the immense popularity of the song "We Are Number One", this Dunderpartrullen remix offers an even catchier rhythm in addition to the already memorable tune. **[Midnaait](https://osu.ppy.sh/u/1506011)**'s mapset capitalizes on this extremely well, offering a wide selection of fun difficulties to play, with contributions from guest mappers Lno and Kantan.
+
+In his flagship Number Oni difficulty, Midnaait shows off the most diverse range of technical snappings. Particularly impressive to note was his stylistic placement of 1/8th doublets throughout - this complemented well with the quirky nature of the song, and gave it an extra "kick" to make it all the more enjoyable.
+
+## osu!catch
+
+[![](/wiki/shared/news/2017-07-22-beatmap-spotlights-june-2017/ultramarine.jpg)](https://osu.ppy.sh/s/275991)
+
+With enjoyable mapping and unique rhythms, **[Zirox](https://osu.ppy.sh/u/200768)** and fellow collaborators **[Ascendance](https://osu.ppy.sh/u/2931883)** and **[Spectator](https://osu.ppy.sh/u/702598)** have created a map that challenges the player's reading skills yet is consistent. Spec's Abyss provides a pleasant reading challenge with Circle Size 5 as well as some custom hitsounds that add to the atmosphere of the map. Rain catches attention with it's lower Approach Rate of 8.7 and dash patterns that are usually uncommon in the current meta. While Cup, Salad and Platter are more of the standard meta, they are still enjoyable to play.
```

Comment:
Spec's -> Spectator's
Circle Size -> circle size
Approach Rate -> approach rate

more of the standard meta -> more standard offerings gameplay wise